static/js/flow/dashboardTypes.js

```diff
@@ -3,11 +3,15 @@ import Decimal from 'decimal.js-light';
 import type { Program } from './programTypes';
 
 // likely to change in very near future
-export type Dashboard = Array<Program>;
+export type Dashboard = {
+  programs:   Array<Program>,
+  is_edx_data_fresh: boolean
```

Comment:
I think this is the type of the object coming from the backend and that is not camelCase